pkg/sessions/persistence/ticket.go

```diff
@@ -30,6 +30,10 @@ type loadFunc func(string) ([]byte, error)
 // a string key for the target of the deletion.
 type clearFunc func(string) error
 
+// lockFunc returns a lock object for a persistent store using a
+// string key
+type lockFunc func(string) sessions.Lock
```

Comment:
Minor observation, not necessarily a problem, I wonder if the name of this implies it locks when it's called? Is there a more explicit name we can use to make sure that's not confusing? Is it just me? (This isn't a blocker just a thought for conversation)

Reply:
I think `initLockFunc` is a better name here.